validation_services/etherscan.php

```diff
@@ -139,7 +139,7 @@ public function transaction_found() {
      */
     public function error() {
         if ( empty( $this->transaction ) ) {
-            return __( 'Could not retrieve transaction information from Etherscan.', 'wc-gateway-nimiq' );
+            return sprintf( __( 'Could not retrieve transaction information from %ss', 'wc-gateway-nimiq' ), 'Etherscan'),
```

Comment:
Must be a semi-colon at the end, not a comma.
Also, double `s`:
```suggestion
            return sprintf( __( 'Could not retrieve transaction information from %s', 'wc-gateway-nimiq' ), 'Etherscan'),
```